### PR TITLE
SDS1 Fix memory overrun in l2pf, l2pm examples.

### PR DESCRIPTION
**Pull request by NandkumarJoshi**
Updated examples, to limit memory under test.

### CODE REVIEW

**Comment by NandkumarJoshi** [comment]
tested this fe-sdk commit to run OK against pcs-kiwi release branch.

File: wit-manifest.json [hunk 1]
```diff
@@ -260,12 +260,12 @@
         "source": "git@github.com:sifive/example-hca-metal.git"
     },
     {
-        "commit": "950b3ac73c3e5a2cf7e7463aa05b144e8616569c",
+        "commit": "ea10676cde77d8417b0b805b32631708f2f72a91",
```
Comment (bsousi5) [inline wit-manifest.json:263]: For both examples, add a comment the purpose of +16 for the address, and use metal api for cflush for consistency. you can put the asm in a comment as an alternate usage.

Comment (NandkumarJoshi) [reply wit-manifest.json]: updated examples with comment, metal API for cache flush.

**Review by bsousi5 — APPROVED** [review]
LGTM

**Pull request merged**